app/api.go

```diff
@@ -111,7 +111,13 @@ func doCreate(enviro env.Project, appJson, rootDir, appName, vendorDir, constrai
 	}
 
 	// Create initial files
-	deps := config.ExtractDependencies(descriptor)
+	var deps []*config.Dependency
+	if descriptor.AppModel != "" {
```

Comment:
can we check version? for example, if >= 1.0. what if someone or something adds app model version to the old file?

Reply:
@mellistibco: Sure. Will add specific check.